Assets/Scripts/API/DFBlock.cs

```diff
@@ -798,7 +800,7 @@ public struct RdbBlockDesc
             public RdbObjectRoot[] ObjectRootList;
 
             /// <summary>List of unknown objects from RdbObjectHeader.</summary>
-            public RdbUnknownObject[] UnknownObjectList;
+            internal RdbUnknownObject[] UnknownObjectList;
```

Comment:
This is something that @Interkarma did with any elements that are in the source data but unused in the API. basically stuff that is unknown or redundant. I converted several public to internals when I did the original world data replacement for buildings since thats the pattern previously used and avoids these fields being serialized - which is important.

If the decision is to make all fields public then I could annotate each field to instruct the serializer, but seems like a lot of work for no benefit. Will let Interkarma decide on thsi one.

Reply:
Yep, TheLacus is right that I've tried to keep DFTFU and DFU mostly separate, with DFU relying on DFTFU and never the other way around. There are unfortunately a few cases where I've done this for speed and ease (aka laziness) but I'd still love to tease them back apart someday if possible.

Let me get back to you on this case once I've worked my way to this PR. Have a lot of stuff to get on top of over next few days.

Reply:
Okay, but I think these are simply fields from original data that should not be public as they're useless or unknown, either way not useful as public API members.

Reply:
My preference is also for the unused/unknown fields to be internal and not public. I can't see any reason for them to be serialised to secondary data or accessed by external mods. If they become useful later, we can make them public then.